src/BloomExe/Edit/EditingView.cs

```diff
@@ -421,11 +405,6 @@ public void AddMessageEventListener(string eventName, Action<string> action)
 			_browser1.AddMessageEventListener(eventName, action);
 		}
 
-		public void UpdateTemplateList()
```

Comment:
Don't know why this was needed...but it feels like the same circumstances would call for updating the button's list?